src/bmpwrite.c

```diff
@@ -333,22 +333,22 @@ void generate_BMP_RGB_data (struct context * context, unsigned char * offset_poi
   if ((context -> source -> color_format & PLUM_COLOR_MASK) == PLUM_COLOR_32)
     data = context -> source -> data;
   else {
-    data = ctxmalloc(context, sizeof *data * context -> source -> width * context -> source -> height);
-    plum_convert_colors(data, context -> source -> data, (size_t) context -> source -> width * context -> source -> height,
-                        PLUM_COLOR_32, context -> source -> color_format);
+    size_t size = (size_t) context -> source -> width * context -> source -> height;
+    data = ctxmalloc(context, sizeof *data * size);
+    plum_convert_colors(data, context -> source -> data, size, PLUM_COLOR_32, context -> source -> color_format);
   }
   size_t rowsize = (size_t) context -> source -> width * 3, padding = 0;
   if (rowsize & 3) {
     padding = 4 - (rowsize & 3);
     rowsize += padding;
   }
-  unsigned char * out = append_output_node(context, rowsize * context -> source -> height);
+  unsigned char * output = append_output_node(context, rowsize * context -> source -> height);
```

Comment:
`output` is a much more common name for this kind of variable elsewhere in the codebase.